py3status/modules/google_calendar.py

```diff
@@ -175,6 +188,15 @@ def _get_credentials(self):
 
         Returns: Credentials, the obtained credential.
         """
+        client_secret_path = os.path.dirname(self.client_secret)
+        auth_token_path = os.path.dirname(self.auth_token)
+
+        if not os.path.exists(auth_token_path):
+            os.makedirs(auth_token_path)
+
+        if not os.path.exists(client_secret_path):
+            os.makedirs(client_secret_path)
+
```

Comment:
We ought to check the files. This results in a large exception message on the bar.
```
1d [chris:~/src/py3status/py3status/modules] igorgrebenkov-addmodule_google_calendar+ ± python3 google_calendar.py
/usr/lib/python3.6/site-packages/oauth2client/_helpers.py:255: UserWarning: Cannot access /home/chris/.config/py3status/google_calendar.auth_token: No such file or directory
  warnings.warn(_MISSING_FILE_MESSAGE.format(filename))
Exception in `test_module` post_config_hook() (InvalidClientSecretsError) module.py line 132.
Traceback
InvalidClientSecretsError: ('Error opening file', '/home/chris/.config/py3status/google_calendar.client_secret', 'No such file or directory', 2)
  File "/home/chris/src/py3status/py3status/module.py", line 132, in prepare_module
    self.module_class.post_config_hook()
  File "google_calendar.py", line 179, in post_config_hook
    self.credentials = self._get_credentials()
  File "google_calendar.py", line 205, in _get_credentials
    flow = client.flow_from_clientsecrets(self.client_secret, SCOPES)
  File "/usr/lib/python3.6/site-packages/oauth2client/_helpers.py", line 133, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/oauth2client/client.py", line 2135, in flow_from_clientsecrets
    cache=cache)
  File "/usr/lib/python3.6/site-packages/oauth2client/clientsecrets.py", line 165, in loadfile
    return _loadfile(filename)
  File "/usr/lib/python3.6/site-packages/oauth2client/clientsecrets.py", line 125, in _loadfile
    exc.strerror, exc.errno)

terminating module test_module
```
We can throw nice string: `missing client_secret`. Look at my recent `volume_status` PR too.

Reply:
Ok, I handle this now, and fixed some of the other issues you raised.